### PR TITLE
Configuring Travis

### PR DESCRIPTION
**Pull request by MarcusLongmuir**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +10,19 @@ cache:
...
+  - go get github.com/mwitkow/go-conntrack/connhelpers
+  - go get github.com/mwitkow/go-grpc-middleware
+  - go get github.com/mwitkow/go-grpc-middleware/logging/logrus
+  - go get github.com/mwitkow/grpc-proxy/proxy
+  - go get github.com/prometheus/client_golang/prometheus/promhttp
+  - go get github.com/spf13/pflag
+  - go get github.com/rs/cors
+  - go get github.com/golang/protobuf/protoc-gen-go
+  - go get github.com/stretchr/testify
+  - npm install
```
Comment (jonnyreeves) [inline .travis.yml]: Do you not need language nodejs in order to get npm?

Comment (MarcusLongmuir) [inline .travis.yml]: Will find out once [Travis outage](https://www.traviscistatus.com) is over 😬

Comment (MarcusLongmuir) [inline .travis.yml]: Assumed that my usage of `go_import_path` probably means that I have to use `language: go`

**Comment by mwitkow** [comment]
legit.

**Comment by MarcusLongmuir** [comment]
Will reopen when it's working on my fork

**Comment by mwitkow** [comment]
LGTM!

**Pull request merged**